lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1486,7 +1486,7 @@ static bool ParseIRGenArgs(IRGenOptions &Opts, ArgList &Args,
 
   if (Args.hasArg(OPT_use_jit))
     Opts.UseJIT = true;
-  
+
```

Comment:
More whitespace changes snuck into this commit. In general, please don't make whitespace changes at all (even good ones) if you're not touching that part of a file.

Reply:
oops yeah. I'll split that out. My editors are pretty crazy about killing trailing whitespace. :)

Reply:
fixed. (removed this file from that commit. I'll do a white space PR cleanup later)